### PR TITLE
Added Rim station take 2

### PR DESCRIPTION
**Pull request by Starbuckss14**
## About the PR

Added Rim station a small ish map for 60 players. My map has lots of nice hiding spots and areas for roleplay.

## Media

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**
:cl:
- add: Added Rim station

Screenshot (13)
Screenshot (14)
Screenshot (15)
Screenshot (16)
Screenshot (17)
Screenshot (18)
Screenshot (19)

### CODE REVIEW

**Comment by lzk228** [comment]
remove fix part from cl

**Review by lzk228 — CHANGES_REQUESTED** [review]
you have some commits that don't supposed to be here

**Comment by lzk228** [comment]
i think you really should read some docs about using github, and then do prs

**Comment by Starbuckss14** [comment]
> remove fix part from cl

Fixed

**Comment by Starbuckss14** [comment]
> you have some commits that don't supposed to be here

Ya I fixed that reverting the edits I made by mistake from before. The files being added are the map and prototype

**Comment by Starbuckss14** [comment]
> i think you really should read some docs about using github, and then do prs

The map and prototype file are included, I don't see the issue. I am not a computer science major

File: Resources/Prototypes/Entities/Objects/Fun/skub.yml [hunk 1]
```diff
@@ -14,4 +14,4 @@
     sound:
       collection: Skub
   - type: UseDelay
-    delay: 2.0
```
Comment (deltanedas) [inline Resources/Prototypes/Entities/Objects/Fun/skub.yml:17]: pro

File: Resources/Prototypes/Entities/Objects/Fun/skub.yml [hunk 2]
```diff
@@ -14,4 +14,4 @@
...
+    delay: 2
```
Comment (lzk228) [inline Resources/Prototypes/Entities/Objects/Fun/skub.yml:17]: that shouldn't be in this pr

File: Resources/Prototypes/Entities/Objects/Fun/toys.yml [hunk 3]
```diff
@@ -21,7 +21,7 @@
       collection: ToyFall
   - type: ItemCooldown
   - type: UseDelay
-    delay: 1.0
+    delay: 1
```
Comment (lzk228) [inline Resources/Prototypes/Entities/Objects/Fun/toys.yml:24]: +

File: Resources/Prototypes/Entities/Objects/Specific/Janitorial/soap.yml [hunk 4]
```diff
@@ -20,7 +20,7 @@
   - type: Item
     sprite: Objects/Specific/Janitorial/soap.rsi
   - type: Slippery
-    paralyzeTime: 2
+    paralyzeTime: 3.5
```
Comment (lzk228) [inline Resources/Prototypes/Entities/Objects/Specific/Janitorial/soap.yml:23]: +

File: Resources/Prototypes/Entities/Objects/Weapons/Melee/e_sword.yml [hunk 5]
```diff
@@ -2,7 +2,7 @@
   name: energy sword
   parent: BaseItem
   id: EnergySword
-  description: A very loud & dangerous sword with a beam made of pure, concentrated plasma. Cuts through unarmored targets like butter.
+  description: A very loud & dangerous sword with a beam made of pure, concentrated plasma. Make those without the highground suffer. Cuts through unarmored targets like butter.
```
Comment (lzk228) [inline Resources/Prototypes/Entities/Objects/Weapons/Melee/e_sword.yml:5]: +

File: Resources/Prototypes/Entities/Objects/Weapons/Melee/e_sword.yml [hunk 6]
```diff
@@ -181,7 +181,7 @@
     litDisarmMalus: 0.7
   - type: MeleeWeapon
     wideAnimationRotation: -135
```
Comment (lzk228) [inline Resources/Prototypes/Entities/Objects/Weapons/Melee/e_sword.yml:183]: +

**Review by lzk228 — CHANGES_REQUESTED** [review]
got it?

File: Resources/Prototypes/Entities/Objects/Fun/skub.yml (hunk 1 above)
Comment (Starbuckss14) [reply Resources/Prototypes/Entities/Objects/Fun/skub.yml]: > pro

Based proskub moment

**Comment by Starbuckss14** [comment]
> Your random changes, found [here](https://github.com/space-wizards/space-station-14/pull/21926/commits/966e76e27e323984ad2da69a3b873ffe80c15546), should be atomized (Moved) to a separate PR if you wish for them to be added, though it seems like these changes are "change for change sake", but that is an entirely seperate discussion. To my understanding you seem to be using github desktop, you can revert commits to your branch, if need be, relatively easily by:
> 
> 1. Navigating to your history tab for your respective branch, in this case `Starbuckss14:master`.
> 2. Finding the respective commit(s) that have caused this issue.
> 3. Right clicking and pressing `Revert Changes In Commit`
> 4. You will then have to push your revert(s) to origin (your primary branch you are working on.)
>    T4CKxOVrI6
> 
> In the future, I suggest creating a new branch on github desktop for any projects you may be undertaking. Currently you are pushing any and all changes to your personal repository to this PR. Create a branch with you relevant map files, and then PR that.

Ok Thank you so much for your help! This fixed the issue! Very based as one might say!

**Comment by Starbuckss14** [comment]
I am happy to report that I have fixed the issue! special thanks to PursuitInAshes!

**Review by lzk228 — COMMENTED** [review]
i personally don't like that here is 2 mimes, 2 clowns, 2 musicians and 2 paramedics for ONLY 60 PLAYERS

about map, in some places too much decals in some too less

**Comment by Starbuckss14** [comment]
This PR is getting closed! The map Rim will remain on my pc if I come back to it or if anyone wants to take over the project. I switched work to my new asteroid map "Phobos." Below are some teasers!
Screenshot (25)
Screenshot (26)
Screenshot (27)

**Pull request closed without merging**